pkg/odo/cli/component/create.go

```diff
@@ -542,7 +539,7 @@ func (co *CreateOptions) Complete(name string, cmd *cobra.Command, args []string
 
 		if util.CheckPathExists(co.DevfilePath) || co.devfileMetadata.devfilePath.value != "" {
 			// Categorize the sections
-			log.Info("Validation")
+			log.Info("Validation for Devfile component")
```

Comment:
Don't change this IMO, it should just stay as `Validation`. No need to specify Devfile. We're trying to avoid using the word Devfile anyways in the UI, since it's actually what's being used "under the hood" / engine, rather than what the user will be directly interacting with.

Reply:
Hi @cdrage , If we keep it just as `Validation` then the logs come out as following if an S2I component is created by default.

![image](https://user-images.githubusercontent.com/7551036/95309832-441a4680-08a9-11eb-9915-b31d71901d40.png)

IMO, some verbosity could actually help in this case. Also, this verbosity will actually fix #4018 

@girishramnani @kadel @adisky @mik-dass you thoughts are welcome as well

Reply:
Currently it looks like this.

![image](https://user-images.githubusercontent.com/7551036/95311680-9197b300-08ab-11eb-931e-6fe9011d7e83.png)

Reply:
Maybe we can revert back to just `Validation` after we get rid of S2I totally ?

Reply:
I think using only `Validation` is ok, explicitly specifying `Validation for devfile component`/ `Validation for S2I component` does not seem very user friendly. To solve the problem in better way  we should use a common spinner ` Validating the component` if one of the s2i or devfile component found just continue with it.
```
$ odo create java
Validation 
 ✓  Validating the component 
```

if no s2i or devfile component found
```
$ odo create java
Validation 
 ✗ No devfile or s2i component found, please run `odo catalog list components` for a list of supported components.
```

Reply:
I agree with @adisky we should keep it as `Validation`. We should not be outputting both s2i and devfile in the output either... 

Having both would indeed confused the user